admin.php

```diff
@@ -7,39 +7,148 @@
+
+    /**
+     * Template folder directory.
+     */
+    const TEMPLATE_DIR = __DIR__ . '/tpl';
+
+    public function getMenuSort()
+    {
+        return self::MENU_SORT;
     }
 
     /**
-     * @return bool true if only access for superuser, false is for superusers and moderators
+     * Admins or higher can access this plugin.
+     * It only has page data that is accessible for admins.
+     * You do not need to be a superuser to access this plugin.
      */
-    public function forAdminOnly() {
+    public function forAdminOnly()
```

Comment:
Why is it for admins only? Every user could potentially add missing titles.

Reply:
This method says like: 
*Do you want to enable this plugin for Admins or for Superadmins only?*

I guess the best way is to also show this for *Admins* as well.